src/main/scala/it/pagopa/interop/catalogmanagement/model/persistence/projection/EServiceCqrsProjection.scala

```diff
@@ -68,7 +72,8 @@ object EServiceCqrsProjection {
           ),
           Updates.push("data.descriptors.$[elem].docs", doc.toDocument)
         )
-    case CatalogItemDocumentDeleted(esId, dId, docId)          =>
+      }
+    case CatalogItemDocumentDeleted(esId, dId, docId)                      =>
```

Comment:
`serviceUrls` in the descriptor needs to be emptied if the document is the interface